Maxis/wwwroot/map/map.component.ts

```diff
@@ -570,175 +588,140 @@ export class MapComponent implements OnInit {
     }
 
     private scrollToBottom(elem: any) {
         elem.scrollTop = elem.scrollHeight;
     }
-
-
-
-
     public latitude: number;
     public longitude: number;
     // public searchControl: FormControl;
 
     @ViewChild("search")
     public searchElementRef: ElementRef;
-    isAvailable = true;
+    isAvailable = true;
 
     constructor(
         private mapsAPILoader: MapsAPILoader,
         private ngZone: NgZone,
         private router: Router, private mapService: MapService,
 
-        //console.log(this.markerTypes),
     ) { }
 
     selectMultiple1: any;
     selectSingle1: any;
     foo: any[];
 
-        ngOnInit() {
-
-            this.form = new FormGroup({});
-            this.form.addControl("selectSingle", new FormControl(""));
-            this.form.addControl("selectMultiple", new FormControl(""));
-            this.form.addControl("selectSingle1", new FormControl(""));
-            this.form.addControl("selectMultiple1", new FormControl(""));
-            this.form.addControl("searchControl", new FormControl(""));
-           // this.getMarker();
-            //this.getDistance();
-            this.getCabelTypes();
-            //this.mapService.getMarker()
-            //    .subscribe((value) => {
-            //        debugger;
-            //        var NEArr = [];
-            //        for (let v in value) {
-            //            var NEList = {
-            //                label: "",
-            //                value:""
-            //            };
-            //            NEList.label = value[v]; 
-            //            NEList.value = value[v];
-            //            NEArr.push(NEList);
-            //        }
-            //        this.markerTypes = NEArr;
-            //    });
-            this.mapService.getCableTypes()
-                .subscribe((value) => {
-                    debugger;
-                    var CAArr = [];
-                    for (let c in value) {
-                        var CAList = {
-                            label: "",
-                            value: ""
-                        };
-                        CAList.label = value[c];
-                        CAList.value = value[c];
-//console.log(c);
-                        CAArr.push(CAList);
+    ngOnInit() {
+
+        this.form = new FormGroup({});
+        this.form.addControl("selectSingle", new FormControl(""));
+        this.form.addControl("selectMultiple", new FormControl(""));
+        this.form.addControl("selectSingle1", new FormControl(""));
+        this.form.addControl("selectMultiple1", new FormControl(""));
+        this.form.addControl("searchControl", new FormControl(""));
+        // this.getMarker();
+        //this.getDistance();
+        this.getCabelTypes();
+        this.getBuilding();
+        this.getCable();
+        this.getLRD();
+                // set google maps defaults
+        this.zoom = 6;
+        this.lat = 4.210484;
+        this.lng = 101.97576600000002;

```

Comment:
get lat, lng, zoom info from api which gets it from a config. **Do not hard code**.